docs/content/en/docs/tasks/cluster/etcd-backup-restore.md

```diff
@@ -0,0 +1,116 @@
+---
+title: "Etcd Backup and Restore"
+linkTitle: "Etcd Backup and Restore"
+weight: 11
+date: 2021-11-04
+---
+
+This page contains steps for backing up a cluster by taking an etcd snapshot, and restoring the cluster from a snapshot. These steps are for an EKS Anywhere cluster provisioned using the external etcd topology (selected by default) and Ubuntu OVAs.
+
+### Use case
+
+EKS-Anywhere clusters use etcd as the backing store. Taking a snapshot of etcd backs up the entire cluster data. This can later be used to restore a cluster back to an earlier state if required. Etcd backups can be taken prior to cluster upgrade, so if the upgrade doesn't go as planned you can restore from the backup.
+
+
+### Backup
+
+Etcd offers built-in snapshot mechanism. You can take a snapshot using the `etcdctl snapshot save` command by following the steps given below. 
+
+1. Login to any one of the etcd VMs
+```
+ssh -i $PRIV_KEY ec2-user@$ETCD_VM_IP
+```
+2. Run the etcdctl command to take a snapshot with the following steps
```

Comment:
```suggestion
2. While logged in as ec2-user in that user's home directory, run the etcdctl command to take a snapshot with the following steps
```
I think those are the assumptions you are making, based on the scp command you run later. In which case, you shouldn't need the chown command and probably don't need the chmod command.

Reply:
@chrisnegus I've seen that to run the etcdctl snapshot command with all the right configuration parameters I had to run it as root, and not as the ec2-user. 
In the next step, scp is run on the local/admin vm using the ec2-user account, that's why I added the chown and chmod commands so there are enough permissions to copy the file to the local machine